plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergConfig.java

```diff
@@ -341,6 +343,19 @@ public double getMinimumAssignedSplitWeight()
         return minimumAssignedSplitWeight;
     }
 
+    public boolean isHideMaterializedViewStorageTableEnabled()
+    {
+        return hideMaterializedViewStorageTableEnabled;
+    }
+
+    @Config("iceberg.materialized-views.hide-storage-tables")
```

Comment:
Please add a test using `iceberg.materialized-views.hide-storage-tables=false`.
We need to ensure that Trino can still read/refresh this kind of materialized views.

Reply:
talked offline that this can be a follow-up.